client/payment-details/transaction-breakdown/index.tsx

```diff
@@ -0,0 +1,244 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { useTimeline } from 'wcpay/data';
+import {
+	Card,
+	CardBody,
+	CardHeader,
+	CardFooter,
+	Flex,
+	FlexItem,
+} from '@wordpress/components';
+import { TimelineItem, TimelineFeeRate } from 'wcpay/data/timeline/types';
+import Loadable, { LoadableBlock } from 'components/loadable';
+import { formatCurrency } from 'multi-currency/interface/functions';
+import { formatFeeType, formatFeeRate } from './utils';
+import { useTransactionAmounts } from './hooks';
+import './style.scss';
+
+interface PaymentTransactionBreakdownProps {
+	paymentIntentId: string;
+}
+
+const PaymentTransactionBreakdown: React.FC< PaymentTransactionBreakdownProps > = ( {
+	paymentIntentId,
+} ) => {
+	const { timeline, isLoading } = useTimeline( paymentIntentId );
+
+	/**
+	 * Right now there is now support for multi-capture in the WooPayments and
+	 * we retrieve information about fees from the first available capture
+	 * event. This should be updated if multi capture becomes reality.
+	 */
+	const captureEvent: TimelineItem | undefined = find(
+		timeline,
+		( item: TimelineItem ) => item.type === 'captured'
+	);
+
+	const transactionAmounts = useTransactionAmounts( captureEvent );
+
+	if (
+		! captureEvent?.transaction_details ||
+		! captureEvent?.fee_rates ||
+		! transactionAmounts
+	) {
+		return null;
+	}
+
+	const { formattedAmount, isMultiCurrency } = transactionAmounts;
```

Comment:
minor: this could be written in a slighlty more condensed way:
```suggestion
	const { formattedAmount, isMultiCurrency } =  useTransactionAmounts( captureEvent );
```